File: src/main/java/com/partymakers/shareparty/data/persistence/expense/entity/ExpenseEntity.java
package com.partymakers.shareparty.data.persistence.expense.entity;



import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@Table(name = "expenses")
@AllArgsConstructor
@Entity
public class ExpenseEntity {
    @Id
    @GeneratedValue
    private long id;
    @Column(name = "name")
    private String name;
    @Column(name = "cost")
    private int    cost;
    @Column(name = "count")
    private double count;
}

File: src/main/kotlin/com/partymakers/shareparty/party/data/dto/ExpenseEntity.kt
package com.partymakers.shareparty.party.data.dto

import com.fasterxml.jackson.annotation.JsonProperty

internal data class ExpenseEntity(
    @JsonProperty("expense_id")
    val id: Long,
    @JsonProperty("name")
    val name: String,
    @JsonProperty("cost")
    val cost: Int,
    @JsonProperty("count")
    val count: Double,
) 